ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -1199,6 +1199,17 @@ tests:
     test:
     - chain: cucushift-sdn-migration-ovn
     workflow: cucushift-installer-rehearse-azure-ipi-sdn
+- as: azure-ipi-ultrassd-disks-p3-f28
+  cron: 56 3 17 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      E2E_RUN_TAGS: '@amd64 and @azure-ipi and @network-ovnkubernetes and not @fips'
```

Comment:
In 4.11, the default network type should be sdn, so it should be `@network-openshiftsdn` instead of `@network-ovnkubernetes`.

Reply:
ok , will change this ? from 4.12 onwards i think ovnkubernetes is default correct ?

Reply:
right, only need to change 4.11.